docs/js/redux-saga.md

```diff
@@ -29,43 +29,47 @@ some examples! (read [this comparison](https://stackoverflow.com/questions/34930
 ## Usage
 
 Sagas are associated with a container, just like actions, constants, selectors
-and reducers. If your container already has a `sagas.js` file, simply add your
-saga to that. If your container does not yet have a `sagas.js` file, add one with
+and reducers. If your container already has a `saga.js` file, simply add your
+saga to that. If your container does not yet have a `saga.js` file, add one with
 this boilerplate structure:
 
 ```JS
 import { take, call, put, select } from 'redux-saga/effects';
 
-// Your sagas for this container
-export default [
-  sagaName,
-];
-
-// Individual exports for testing
-export function* sagaName() {
-
+// Root saga
+export default function* sagaName() {
+  // if necessary, start multiple sagas at once with `all` 
+  yield all([
+    helloSaga(),
```

Comment:
From what I understand of the saga API, it could just be

```js
export default function* sagaName() {
  yield all([
    takeLatest(LOAD_REPOS, getRepos),
    takeLatest(LOAD_USERS, getUsers),
  ]);
};
```

Much more concise than putting each of those in a `function*`.